app/models/article.rb

```diff
@@ -88,16 +88,23 @@ class Article < ApplicationRecord
   scope :published, -> { where(published: true) }
   scope :unpublished, -> { where(published: false) }
 
+  scope :latest_published_admin_thread_with, lambda { |tag_name|
+    published.
+      where(user_id: SiteConfig.staff_user_id).
+      order("published_at ASC").
```

Comment:
nitpicking here: `published_at: :desc` :D

Reply:
Did you meant to say `:asc`? I did change it to `desc` anyway though to keep it consistent with one other scope.